input.go

```diff
@@ -10,12 +10,35 @@ var (
 	MOVE    = Action(0)
 	PRESS   = Action(1)
 	RELEASE = Action(2)
+	NEUTRAL = Action(99)
 	SHIFT   = Modifier(0x0001)
 	CONTROL = Modifier(0x0002)
 	ALT     = Modifier(0x0004)
 	SUPER   = Modifier(0x0008)
 )
 
+// MouseButton corresponds to a mouse button.
+type MouseButton int
+
+// Mouse buttons
+const (
+	MouseButton1      MouseButton = 0
+	MouseButton2      MouseButton = 1
+	MouseButton3      MouseButton = 2
+	MouseButton4      MouseButton = 3
+	MouseButton5      MouseButton = 4
+	MouseButton6      MouseButton = 5
+	MouseButton7      MouseButton = 6
+	MouseButton8      MouseButton = 7
+	MouseButtonLast   MouseButton = 7
+	MouseButtonLeft   MouseButton = 1
+	MouseButtonRight  MouseButton = 2
+	MouseButtonMiddle MouseButton = 3
+)
+
+// these values are remapped during engin_glfw init()
+// so I suppose these are the default mappings for engi_js
+// TODO: can someone confirm this by removing this comment and setting a proper one ?
```

Comment:
Could we define them as

````go
var (
Dash Key
Apostrophe Key
)
```

in this file, and then give them values in `engi_js.go` and `engi_glfw.go`?